src/main/services/bs-installer.service.ts

```diff
@@ -79,9 +79,13 @@ export class BSInstallerService {
 
         await ensureDir(this.installLocationService.versionsDirectory);
 
+        const isLinux = process.platform === 'linux';
+        const exePath = this.getDepotDownloaderExePath();
+        const args = DepotDownloader.buildArgs(depotDownloaderOptions);
+
         return new DepotDownloader({
-            command: this.getDepotDownloaderExePath(),
-            args: DepotDownloader.buildArgs(depotDownloaderOptions),
+            command: isLinux ? 'dotnet' : this.getDepotDownloaderExePath(),
+            args: isLinux ? [exePath, ...args] : [exePath],
```

Comment:
plus, the exePath is already used as `command` on windows normaly

Reply:
Meant to have that as `args`, not `exePath`.

I was having issues with Wine not finding a required library.
Instead of running it through Wine, it's easier to run it natively through dotnet to ensure compatibility.

Reply:
Yeah that good to use `dotnet` command to start the exe on linux 👌 
But on windows the the command must stay the path of the exe, and not have it in the args

Reply:
Yeah that was a mistake, I meant to write args, not sure how I fumbled that :sweat_smile: